spec/spec_helper.rb

```diff
@@ -98,4 +98,3 @@
 end
 
 require "rspec/json_expectations"
-require 'webmock/rspec'
```

Comment:
you may want to mention this change in the PR description in case we're ever trying to track down changes ;)